### PR TITLE
REST docs for linkrequest API

### PR DESCRIPTION
**Pull request by alavers**
screen shot 2017-11-14 at 4 24 32 pm

### CODE REVIEW

File: slate/source/includes/rest/core/_app_user.md [hunk 1]
```diff
@@ -565,6 +565,77 @@ Users can begin their communication over a third party channel such as Facebook
...
+> Request:
+
+```shell
+curl https://api.smooch.io/v1/apps/5963c0d619a30a2e00de36b8/appusers/deb920657bbc3adc3fec7963/linkrequest?integrationIds=59dbd737f294ea1fa2734829,59fa19888b1dd03638d2dd54,5a00d5b3002ef2a871aaef98 \
+     -X GET \
+     -H 'content-type: application/json' \
+     -H 'authorization: Bearer your-account-jwt'
+```
+```js
+smooch.appUsers.getLinkRequests('5963c0d619a30a2e00de36b8', 'deb920657bbc3adc3fec7963', '59dbd737f294ea1fa2734829,59fa19888b1dd03638d2dd54,5a00d5b3002ef2a871aaef98')
```
Comment (mspensieri) [inline slate/source/includes/rest/core/_app_user.md]: I thought smooch core was accepting an array?

File: slate/source/includes/rest/core/_app_user.md [hunk 2]
```diff
@@ -565,6 +565,77 @@ Users can begin their communication over a third party channel such as Facebook
...
+
+> Response:
+
+```
+200 OK
+```
+
+```
+{
+	"linkRequests": [
```
Comment (mspensieri) [inline slate/source/includes/rest/core/_app_user.md]: nit: this indentation looks rather huge

File: slate/source/includes/rest/core/_app_user.md [hunk 3]
```diff
@@ -565,6 +565,77 @@ Users can begin their communication over a third party channel such as Facebook
...
+		}
+	]
+}
+```
+
+| Query Parameter    | Description                                              |
+|--------------------|----------------------------------------------------------|
+| **integrationIds** | Comma separated list of [integration](#integrations) ids |
+
+A user who begins their conversation via one of the Smooch SDKs may wish to transfer the conversation to different preferred channel. For example, a user may leave you a message on your website and ask to be notified by Facebook as soon as you respond.
```
Comment (mspensieri) [inline slate/source/includes/rest/core/_app_user.md]: to *a* different preferred channel

File: slate/source/includes/rest/core/_app_user.md (hunk 1 above)
Comment (alavers) [reply slate/source/includes/rest/core/_app_user.md]: oop. yes indeed it does

**Pull request merged**